<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.1.2 (Python 3.8.0 on win32)" generated="20200127 08:20:04.844" rpa="false">
<suite id="s1" name="Close All.Robot" source="C:\Users\pmarchenko\Documents\Ptestpy\RobotTests\testCases\close_all.robot.robot">
<test id="s1-t1" name="MyTestCase">
<kw name="Open Browser" library="SeleniumLibrary">
<doc>Opens a new browser instance to the optional ``url``.</doc>
<arguments>
<arg>http://demowebshop.tricentis.com/register</arg>
<arg>chrome</arg>
</arguments>
<msg timestamp="20200127 08:20:08.280" level="INFO">Opening browser 'chrome' to base url 'http://demowebshop.tricentis.com/register'.</msg>
<kw name="Capture Page Screenshot" library="SeleniumLibrary">
<doc>Takes a screenshot of the current page and embeds it into a log file.</doc>
<msg timestamp="20200127 08:20:08.336" level="INFO">Cannot capture screenshot because no browser is open.</msg>
<status status="PASS" starttime="20200127 08:20:08.336" endtime="20200127 08:20:08.336"></status>
</kw>
<msg timestamp="20200127 08:20:08.514" level="FAIL">WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home
</msg>
<status status="FAIL" starttime="20200127 08:20:08.280" endtime="20200127 08:20:08.515"></status>
</kw>
<status status="FAIL" starttime="20200127 08:20:08.279" endtime="20200127 08:20:08.515" critical="yes">WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home
</status>
</test>
<status status="FAIL" starttime="20200127 08:20:04.850" endtime="20200127 08:20:08.517"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1">Critical Tests</stat>
<stat pass="0" fail="1">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" id="s1" name="Close All.Robot">Close All.Robot</stat>
</suite>
</statistics>
<errors>
</errors>
</robot>
